nix/packages.nix

```diff
@@ -18,7 +18,23 @@
     {
       packages = with flake.packages; {
         inherit "ytxp-plutarch:lib:ytxp-plutarch" "ytxp-plutarch:test:serialization" "ytxp-plutarch:exe:write-config";
+        doc = import inputs.combine-haddock.outPath { inherit pkgs lib; } {
+          cabalProject = pkgs.ytxp-plutarch;
+          targetPackages = [
+            "ytxp-plutarch"
+          ];
+          prologue = ''
+            = Plutarch Documentation
+            Documentation of Plutarch /and/ Documentation of Plutus libraries.
```

Comment:
@albertoefg We forgot to change this text
@albertodvp What do you want here? This is the text that appears in the home page
![image](https://github.com/mlabs-haskell/ytxp-lib/assets/2318843/994d8329-ae32-4d14-bec4-7a9aeccb9b8b)

Reply:
@nazrhom Done, we can try merging and see if github-pages works